bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -177,6 +176,11 @@ where
 		}))
 	});
 
+	io.add_sync_method("bitacross_getScheduledEnclaveList", move |_: Params| {
```

Comment:
should this be behind the "development" feature flag ?

Reply:
It should be OK for production too as it's meant to be public information

Reply:
I see, if this is not secret info why do we seal it?

Reply:
To prevent it from being modified (intentionally or unintentionally) by worker operators. So the public should have read access but no write access.

I mean even if you can't read it, you should be able to infer the content just by checking the parachain tx and tee-worker code : )

Reply:
Got it 👍🏼